charts/signon-resources/templates/bootstrap-job.yaml

```diff
@@ -126,6 +126,14 @@ spec:
                   "home_uri": "https://hmrc-manuals-api.eks.{{ .Values.govukEnvironment }}.{{ .Values.govukDomainExternal }}",
                   "redirect_uri": "https://hmrc-manuals-api.eks.{{ .Values.govukEnvironment }}.{{ .Values.govukDomainExternal }}/auth/gds/callback",
                   "permissions": ["signin"]
+                },
+                "router-api-eks": {
+                  "name": "Router API [EKS]",
+                  "secret_name": "signon-app-router-api",
```

Comment:
Not sure if yo uneed to - but some others have '-eks' appended to the secret name